ui/src/Components/ProfilePage.jsx

```diff
@@ -86,16 +138,15 @@ export default class ProfilePage extends React.Component {
                   ) : <div/>}
                   <tr>
                     <td>Id Verified</td>
-                    {this.state.profile.result.gov_id ? <td>Verified</td> : <td>Not Verified</td> }
+                  {/* must be updated in the GET ticket to import the state directly from the DB */}
+                    <td>{this.state.profile.result.app_status}</td>
```

Comment:
@christinajiyunlee this was one of your requirements for your ticket, but I needed this to be displayed for testing purposes

Reply:
✔

Reply:
Sounds good. Feel free to modify this in your ticket. But there is no separate check for ID, its all captured in the flow. i.e there is no ID verified specifically, its just either application is verified or still pending